File: app/src/main/java/com/marceljurtz/lifecounter/models/ViewHelper.java
package com.marceljurtz.lifecounter.models;

import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.Nullable;

public class ViewHelper {

    @Nullable
    public static View findFirstViewByTag(ViewGroup root, String tag) {

        final int childCount = root.getChildCount();
        for (int i = 0; i < childCount; i++) {
            final View childView = root.getChildAt(i);

            if (childView.getTag() != null && childView.getTag().toString().equals(tag)) {
                return childView;
            }
        }

        return null;
    }

    public static String lblDescriptionTag = "description";
    public static String lblAtkTag = "atk";
    public static String lblDefTag = "def";
}

File: app/src/main/java/com/marceljurtz/lifecounter/models/ViewHelper.kt
package com.marceljurtz.lifecounter.models

import android.view.View
import android.view.ViewGroup

object ViewHelper {

    var lblDescriptionTag = "description"
    var lblAtkTag = "atk"
    var lblDefTag = "def"

    fun findFirstViewByTag(root: ViewGroup, tag: String): View? {

        val childCount = root.childCount
        for (i in 0 until childCount) {
            val childView = root.getChildAt(i)

            if (childView.tag != null && childView.tag.toString() == tag) {
                return childView
            }
        }

        return null
    }
}
